jsm.go

```diff
@@ -611,6 +627,26 @@ type StreamLister struct {
 	pageInfo *apiPaged
 }
 
+// StreamsInfo returns a receive only channel to iterate on the streams.
+func (js *js) StreamsInfo(ctx context.Context) <-chan *StreamInfo {
+	ach := make(chan *StreamInfo)
```

Comment:
In case context is nil here, I think we should create one that defaults to the value of `js.wait`